src/main/java/com/grafana/opentelemetry/GrafanaOtlpConfig.java

```diff
@@ -19,7 +19,7 @@ public Map<String, String> resourceAttributes() {
 
     @Override
     public String url() {
-        return p.getEndpoint().orElse(OtlpConfig.DEFAULT.url());
+        return p.getEndpoint().map(s -> s + "/v1/metrics").orElse(OtlpConfig.DEFAULT.url());
```

Comment:
Wondering if the class name should reflect that the config are for metrics only?